eclipse-theia/theia
# plugin: reset `validateInput` on subsequent calls

#### What it does


Fixes: https://github.com/eclipse-theia/theia/issues/10393

The pull-request fixes an issue where the `validateInput` handler is not reset on subsequent `showInput` calls leading to the input being incorrectly validated. The fix ensures that if a `showInput` omits `validateInput` that it will reset the cached handler.

#### How to test


1. include the following [show-input](https://github.com/vince-fugnitto/vscode-show-input/releases/download/v0.0.1/vscode-showinput-0.0.1.vsix)
2. confirm that the steps from #10393 work well - the `validateInput` should not be preserved on subsequent `showInput` calls

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
